firebase.json

```diff
@@ -14,6 +14,13 @@
     },
     "ui": {
       "enabled": true
+    },
+    "database": {
+      "port": 9000
     }
+  },
+  "firestore": {
+    "rules": "firestore.rules",
+    "indexes": "firestore.indexes.json"
```

Comment:
Looks like `rules` is for permissions, what is `indexes` for?

Reply:
it's for what fields firebase should store all the info for so it can sort on that key

Reply:
Ok.. why is it necessary now when it wasn't before?

Reply:
it's because I initialized the command line tools. You can also set all this stuff in the gui, but once you're deploying functions, you can also deploy the whole database from the command line. So you have the files locally